src/combine.cpp

```diff
@@ -5847,63 +5848,28 @@ static void handle_oc_property_decl(chunk_t *os)
          // now.
          while (next != nullptr && next->type != CT_PAREN_CLOSE)
          {
-            if (next->type == CT_WORD)
+            if (next->type == CT_OC_PROPERTY_ATTR)
```

Comment:
Yes. It's part of the method handle_oc_property_decl, which will be called only for CT_OC_PROPERTY